src/test/java/opinion/sentences/SentenceParserTest.java

```diff
@@ -1,13 +1,41 @@
 package opinion.sentences;
 
 import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
+/**
+ * Tests behavior of the SentenceParser class
+ */
 public class SentenceParserTest {
 
     @Test
-    public void firstTest() {
-        Assert.assertTrue(true);
+    /**
```

Comment:
nit: no need to leave comments on test methods unless they add some explanation that isn't obvious.

Also, try to put javadoc commnets before the annotation (`@Test` in this case, or `@Nullable` for example)